src/ansys/dpf/post/simulation.py

```diff
@@ -601,6 +601,26 @@ def _build_selection(
             return selection
         else:
             selection = Selection(server=self._model._server)
+
+        if isinstance(skin, bool):
+            has_skin = skin
+        else:
+            has_skin = len(skin) > 0
```

Comment:
@janvonrickenbach what is the use-case for `skin` to be a list of `int`?

Reply:
It is an element selection. The skin is extracted for these elements. This was already part of the api.